services/QuillLMS/lib/tasks/schools.rake

```diff
@@ -6,22 +6,11 @@ require 'open-uri'
 # This code is taken from the Literacy Design Collaborative (ldc.org) and then modified.
 def nces_grade_level_mapping(nces_grade)
   case nces_grade
-  when 'PK'
-      0
-  when 'KG'
-      0
-  when 'M'
-      nil
-  when 'N'
-      nil
-  when 'UG'
-      nil
-  when 'AE'
-      nil
-  when '13'
-      12
+  when 'PK', 'KG' then 0
```

Comment:
Another case statement that's probably better as a hash.

Reply:
I'll probably just leave this one as-is since it's a rake task.